src/command/tools/cmd.ts

```diff
@@ -81,7 +82,6 @@ Use 'quarto tools' with no arguments to show the status of all tools.`,
           );
         }
         break;
-      default:
```

Comment:
I think you should consider handling the `default` case of any unrecognized command (e..g `quarto tools foobar`). Currently that just does nothing and prior to this change that would run the `list` command. Both don't seem right - I think an error is warranted in this case since the user is providing a command that we don't know how to fulfill.